### PR TITLE
Document unlocking of exercises

### LINKED ISSUE #332: Enhancement: Focusing attention on the Learning vs Practice mode switch
There is an existing section in [the FAQ mentioning that Learning mode is the default on Exercism v3](https://github.com/exercism/docs/blob/main/using/faqs.md#why-have-my-unlocked-exercises-in-the-old-site-become-locked). It looks like new users (and yours truly) may miss this vital piece of information, or that "Disable Learning Mode" will effectively enable Practice mode.

Two possible improvements are coming to mind:
- Add a quick reference to this fact in [Getting Started](https://github.com/exercism/docs/blob/main/using/getting-started.md#learn-a-programming-language), i.e. that you start in the limited "Learning Mode" and, in case you already have experience in the track, you can disable it and switch to Practice mode which remotes the prerequisite limitation.
- Add an h2 section in the FAQ mentioning "I already have experience in a language. Is it possible to unlock all material and skip the easier lessons and exercises?"

CC @taiyab

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes https://github.com/exercism/docs/issues/332

### CODE REVIEW

File: building/product/unlocking-exercises.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Exercises are unlocked differently based on the type of track:
+
+1. Tracks without Learning Exercises: all Practice Exercises are unlocked from the start.
+2. Tracks with Learning Exercises: complete Learning Exercises to unlock further Learning and Practice Exercises.
+
+## Learning Mode vs Practice Mode
+
+Requiring Learning Exercises to be completed to unlock more exercises is called Learning Mode (which was introduced with Exercism 3.0).
+Learning Mode is the default for any user that joins a track with Learning Exercises, but also for any users from the old site.
+Practice Mode is the opposite, where all exercises are unlocked from the start.
```
Comment (SaschaMann) [inline building/product/unlocking-exercises.md:12]: This only seems to apply to practice exercise, not all exercises, unless that's a bug or I'm missing something.

Comment (ErikSchierboom) [reply building/product/unlocking-exercises.md:12]: I've just tested this locally, but to me all exercises become unlocked. Not for you?

Comment (SaschaMann) [reply building/product/unlocking-exercises.md:12]: No, if I switch to Practice Mode on the Swift track, the number of exercises is reduced and none of the learning exercises are listed. But then that appears to be a bug so we should figure that out elsewhere.

Comment (ErikSchierboom) [reply building/product/unlocking-exercises.md:12]: Hmmm, weird! Thanks for reporting

**Pull request merged**